src/components/MediaPlayer/VideoJS/VideoJSPlayer.js

```diff
@@ -277,6 +279,27 @@ function VideoJSPlayer({
           volumeIndex + 1
         );
       }
+      /**
+       * Volume panel display on desktop browsers:
+       * For audio: volume panel is inline with a sticky volume slider
+       * For video: volume panel is not inline.
+       * For mobile devices the player uses MuteToggle for both audio
+       * and video.
+       */
+      if (!IS_MOBILE) {
+        controlBar.removeChild('volumePanel');
+        if (!isVideo) {
+          controlBar.addChild('volumePanel', { inline: true }, volumeIndex);
+        } else {
+          controlBar.addChild('volumePanel', { inline: false }, volumeIndex);
+        }
```

Comment:
Does this duplicate the change in MediaPlayer.js or is it needed to place the volume control in the proper order of controls?  If the later, could this be simplified to match the config in MediaPlayer.js?
```suggestion
        controlBar.addChild('volumePanel', { inline: !isVideo }, volumeIndex);
```

Reply:
It's the later.